README.md

```diff
@@ -399,6 +400,24 @@ render((
         uiSchema={uiSchema} />
 ), document.getElementById("app"));
 ```
+### Array items ordering
+
+Array items are sortable by default. The `uiSchema` object spec allows you to disable ordering:
+
+```jsx
+const schema = {
+  type: "array",
+  properties: {
+    type: "string"
+  }
+};
+
+const uiSchema = {
+  "ui:options":  {
+    sortable: false
+  }
```

Comment:
I think that @n1k0 took a different approach in #346 in order to introduce widget options: 
```js
const uiSchema = {
  "ui:widget": {
    component: "checkboxes",
    options: {
      inline: true
    }
  }
};
```

I think that both approaches are valid :) I'll let you figure out pros and cons for each ;)